guide/daily-spending-wallet/backup-and-recovery/recovery.md

```diff
@@ -33,47 +33,395 @@ Illustration sources:
+   layout = "shadow"
+   caption = "App searches for the cloud backup."
+   alt-text = "Screen shows an animation while the cloud backup is found"
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restoring.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restoring@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restoring@2x.png"
+   layout = "shadow"
+   caption = "Wallet restores from cloud backup."
+   alt-text = "Screen shows an animation while the wallet is being restored"
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restored.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restored@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restored@2x.png"
+   layout = "shadow"
+   caption = "Wallet is finished being restored."
+   alt-text = "Screen showing that the wallet has been restored."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-finish.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-finish@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-finish@2x.png"
+   layout = "shadow"
+   caption = "The wallet has been successfully restored."
+   alt-text = "Screen showing a home screen with funds"
+   width = 250
+   height = 541
 %}
 
 </div>
 
-### Restore from an automatic cloud backup
+## Recovering with a manual backup
+
+This involves a few more steps, but should work just as well as the auto cloud backup. This works great for [basic recovery scenarios](#basic-recovery). In the manual restore, the primary difference is that the app asks the user to type in their recovery phrase. It will attempt to automatically pull their lightning channel state from cloud backup.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual@2x.png"
+   layout = "shadow"
+   caption = "User selects \"advanced\" from the restore screen, and is taken here."
+   alt-text = "Screen showing description of manual restore"
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-recovery-phrase-progress.png"
```

Comment:
We should add auto-complete here, as it's enough to enter the first few characters usually.

Reply:
Let's discuss this here before I make another commit. My initial thought is that it should be separate and distinct from the OS level autocomplete. The whole advantage is that it should pull from the limited 2048 word BIP39 dictionary.

We can show one screen where we use autocorrection to prevent an error. The below is similar to the existing screen, but we are instead detecting if the word is wrong before the user even finishes typing it. There's no `poi` words after `point` in the dict, so the app can assume `poil` is a typo.

![restore-manual-recovery-phrase-error](https://user-images.githubusercontent.com/2414177/184961861-4aa38074-aac9-47b2-8f30-b8351e4926da.png)

Then we can show an example of where autocomplete prevents the user from having to type in a long word. In this case, `hospital`, because there are no other `hosp` words in the dict.

![restore-manual-recovery-phrase-progress](https://user-images.githubusercontent.com/2414177/184962231-01df735e-fe4f-483b-9661-e7a879e4fd33.png)

With the above picture, I was thinking about how the user takes advantage of the autocomplete. Do they just tap the word to have it fill automatically? Would they know to do this? What if the autocorrect stood out more? So I made the below image, which would be a little popup that shows the closest guess. The popup is the same color as the primary button, so I think it would stand out as something you interact with.

![restore-manual-recovery-phrase-progress-1](https://user-images.githubusercontent.com/2414177/184962465-09f4365e-67cd-4df5-b71c-a62df059043a.png)

Another option might be to have a custom autocomplete bar above the keyboard, but that might look kind of tacky in place of the OS autocomplete.

Thoughts?

Reply:
I like the OS autocomplete, as long as it only uses words from the word list that is. Also its only required to enter the first 4 letters for each entry, could mention this.

Reply:
Stephen and I discussed yesterday. Option 2 seems the way to go, as OS autocompletes don't allow for custom dictionaries. URL bars also have a similar approach to option 2.

Reply:
@Bosch-0 I decided against the OS autocomplete. It's doesn't matter to me if feeding the OS autocomplete a custom dictionary is possible or not, because the user may not understand this. Seeing words from their recovery phrase start appearing in the autocomplete might feel somewhat alarming, like the phone is tracking their bitcoin activity or something.

Instead, I added something that I hope feels more familiar to how autocomplete would work in a mobile web browser.

Reply:
I like the new design you went for, except for the light blue, doesn't seem to go with everything else.